### PR TITLE
0.20.0-medallia-0

### PR DESCRIPTION
**Pull request by NDonatucci**

### CODE REVIEW

**Review by malozanoff — APPROVED** [review]
LGTM

File: src/main/java/org/apache/aurora/scheduler/TaskStatusHandlerImpl.java [hunk 1]
```diff
@@ -36,7 +36,10 @@
 import org.apache.aurora.scheduler.state.StateManager;
 import org.apache.aurora.scheduler.stats.CachedCounters;
 import org.apache.aurora.scheduler.storage.Storage;
+import org.apache.aurora.scheduler.storage.Storage.MutableStoreProvider;
```
Comment (lspir) [inline src/main/java/org/apache/aurora/scheduler/TaskStatusHandlerImpl.java]: Do we need these three imports?

Comment (NDonatucci) [reply src/main/java/org/apache/aurora/scheduler/TaskStatusHandlerImpl.java]: No, I'll remove them

File: src/test/java/org/apache/aurora/scheduler/mesos/MesosTaskFactoryImplTest.java [hunk 2]
```diff
@@ -23,16 +24,11 @@
...
-import org.apache.aurora.gen.Container;
-import org.apache.aurora.gen.DockerContainer;
-import org.apache.aurora.gen.DockerImage;
-import org.apache.aurora.gen.DockerParameter;
-import org.apache.aurora.gen.Image;
-import org.apache.aurora.gen.MesosContainer;
-import org.apache.aurora.gen.ServerInfo;
-import org.apache.aurora.gen.TaskConfig;
+
+import org.apache.aurora.gen.*;
```
Comment (lspir) [inline src/test/java/org/apache/aurora/scheduler/mesos/MesosTaskFactoryImplTest.java]: nit: maybe we can remove the import with a wildcard, so we can easily see which imports are added by us

**Review by lspir — APPROVED** [review]

File: src/test/java/org/apache/aurora/scheduler/mesos/MesosTaskFactoryImplTest.java (hunk 2 above)
Comment (NDonatucci) [reply src/test/java/org/apache/aurora/scheduler/mesos/MesosTaskFactoryImplTest.java]: Done

**Pull request merged**